tee-worker/ts-tests/parachain-interfaces/augment-api-tx.ts

```diff
@@ -1622,19 +1622,27 @@ declare module '@polkadot/api-base/types/submittable' {
                 [AccountId32, CorePrimitivesKeyAesOutput, H256]
             >;
             /**
-             * Link an identity with the given validation data
+             * Link an identity with given network types and validation data.
```

Comment:
These changes seem to be from last commit of `dev` branch - maybe you need to re-merge it again?

Reply:
This should be the correct version according to:
https://github.com/litentry/litentry-parachain/blob/ae3cfa4f0d1ad359d1aaf468adf86c741f295949/pallets/identity-management/src/lib.rs#L193-L214